### PR TITLE
[BugFix] query tracking_url via leader FE

### PR DESCRIPTION
**Pull request by rickif**
Why I'm doing:
The follower FE has no information about routine load job error logs, and we get an empty result if we query `information_schema.load_tracking_logs`.
What I'm doing:
Let all `information_schema.load_tracking_logs` queries query error logs information on leader FE.
Fixes https://github.com/StarRocks/StarRocksTest/issues/5253

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java [hunk 1]
```diff
@@ -1407,6 +1407,10 @@ public PlanFragment visitPhysicalSchemaScan(OptExpression optExpression, ExecPla
                 throw UnsupportedException.unsupportedException("load_tracking_logs must specify label or job_id");
             }
 
+            if (scanNode.getTableName().equalsIgnoreCase("load_tracking_logs")) {
+                scanNode.setFrontendIP(GlobalStateMgr.getCurrentState().getNodeMgr().getLeaderIp());
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java:1411]: rpc port may be different

Comment (rickif) [reply fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java]: Updated

**Comment by rickif** [comment]
@mergifyio rebase

File: fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java [hunk 2]
```diff
@@ -1411,6 +1412,12 @@ public PlanFragment visitPhysicalSchemaScan(OptExpression optExpression, ExecPla
                 throw UnsupportedException.unsupportedException("load_tracking_logs must specify label or job_id");
             }
 
+            if (scanNode.getTableName().equalsIgnoreCase("load_tracking_logs")) {
+                Pair<String, Integer> ipPort = GlobalStateMgr.getCurrentState().getNodeMgr().getLeaderIpAndHttpPort();
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java:1416]:
```suggestion
                Pair<String, Integer> ipPort = GlobalStateMgr.getCurrentState().getNodeMgr().getLeaderIpAndRpcPort();
```

**Review by wyb — APPROVED** [review]

**Review by meegoo — APPROVED** [review]

**Comment by rickif** [comment]
@mergifyio backport branch-3.1

**Pull request merged**